testsuite/model/src/test/java/org/keycloak/testsuite/model/session/UserSessionProviderModelTest.java

```diff
@@ -93,20 +93,6 @@ public void createEnvironment(KeycloakSession s) {
 
     @Override
     public void cleanEnvironment(KeycloakSession s) {
-        RealmModel realm = s.realms().getRealm(realmId);
```

Comment:
I introduced an issue for this: https://github.com/keycloak/keycloak/issues/19463